index.html

```diff
@@ -0,0 +1,241 @@
+                                    <br> to students of all ages. The importance of an online school like SmartEd Online cannot be 
+                                    <br> overstated. It provides students with the flexibility to learn at their own pace, which is especially 
+                                    <br> important for those who have other commitments such as work or family.
+                                </p>
+                                <div class="d-flex justify-content-start mt-5 align-items-center">
+                                    <p class="mt-5 me-5 ms-2 text-justify text-uppercase start-now"> Start your journey now </p>
+                                    <a href="#" class="btn btn-lg mt-2 me-3 learn-btn">Enroll Now</a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- end banner-->
+    <!-- COURSE FEATURES -->
+    <section id="course-features" class="py-5"> 
+        <h2 class="h2 d-flex justify-content-center ">Course Features</h2>
+        <span class="mt-3 d-flex align-content-center span"></span>
+        <div class="container">
+          <div class="row">
+            <div class="col-md-12">
+              <div class="row p-lg-5">
+                <div class="col-md-3">
+                  <div class="feat-box p-3 pb-lg-5 ">
+                    <div class="feat-img d-flex justify-content-lg-center ">
+                      <img src="images/icons/1.png" alt="">
+                    </div>
+                    <h3 class="text-justify text-center pt-3"> Custom Paths</h3>
+                    <p class="text-justify text-center pt-lg-2"> Create your own path. Decide what you want to study and in what context.</p>
+                  </div>
+                </div>
+                <div class="col-md-3 ">
+                  <div class="feat-box p-3 pb-lg-5">
+                    <div class="feat-img d-flex justify-content-center ">
+                      <img src="images/icons/2.png" alt="">
+                    </div>
+                    <h3 class="text-justify text-center pt-3 ">Master Classes</h3>
+                    <p class="text-justify text-center pt-2"> Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet consectetur.</p>
+                  </div>
+                </div>
+                <div class="col-md-3 ">
+                    <div class="feat-box p-3">
+                      <div class="feat-img d-flex justify-content-center">
+                        <img src="images/icons/5.png" alt="">
+                      </div>
+                      <h3 class="text-justify text-center pt-3">Advanced Evaluation</h3>
+                      <p class="text-justify text-center pt-2"> Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet consectetur.</p>
+                    </div>
```

Comment:
Nice job adding the cards for the course feature section😉 but let's ensure to stick to the design by making the cards 5 in number as shown in the design👍 as it is required that we stick to the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) but personalize it😉

<img width="605" alt="image" src="https://user-images.githubusercontent.com/64643995/233476104-93a829ce-588f-4dd2-9004-c3c5bb334f43.png">

Kindly add one more card to the section to make your project match the[ design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)👍.